### PR TITLE
Add ZH Doc: Configure WeCom Notification

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

**Comment by benjaminhuo** [comment]
@wanjunlei pls help to review this PR

**Comment by Felixnoo** [comment]
@wanjunlei Hi, please help review this PR when you are available, thanks.

**Comment by Felixnoo** [comment]
@wanjunlei Hi, please help review this PR when you are available. It's great that there's a preview link available now.

**Review by wanjunlei — APPROVED** [review]

File: content/zh/docs/cluster-administration/platform-settings/notification-management/configure-wecom.md [hunk 1]
```diff
@@ -1,10 +1,119 @@
...
+
+   ![enter-dept-name](/images/docs/zh-cn/cluster-administration/platform-settings/notification-management/configure-wecom/enter-dept-name.png)
+
+3. 创建部门后，您可以点击右侧的**添加成员**、**批量导入**或**从其他部门移入**来添加成员。添加成员后，点击该成员进入详情页面，查看其帐号。
+
+   ![add-member](/images/docs/zh-cn/cluster-administration/platform-settings/notification-management/configure-wecom/add-member.png)
+
+   ![member-account](/images/docs/zh-cn/cluster-administration/platform-settings/notification-management/configure-wecom/member-account.png)
+
+4. 您可以点击`测试二组`右侧的 <img src="/images/docs/zh-cn/cluster-administration/platform-settings/notification-management/configure-wecom/three-dots.png" height="20px">来查看其部门 ID。
```
Comment (Sherlock113) [inline content/zh/docs/cluster-administration/platform-settings/notification-management/configure-wecom.md:57]: Add a space between the thumbnail and `来查看其部门 ID`.

**Pull request merged**